doc/source/tutorial/linalg.rst

```diff
@@ -21,7 +21,7 @@ scipy.linalg vs numpy.linalg
 .. TODO: replace numpy.linalg HTML link with `numpy.linalg` once NumPy updates doc
 
 :mod:`scipy.linalg` contains all the functions in
-`numpy.linalg <https://www.numpy.org/devdocs/reference/routines.linalg.html>`__.
+:doc:`numpy.linalg <numpy:reference/routines.linalg>`.
```

Comment:
Same sort of thing as above. Not necessarily a problem, but is it what we want? I ask because I think we are trending towards making scipy.org more about the SciPy package rather than the ecosystem, and so I'd think we'd want to direct users toward numpy.org for NumPy documentation.

Reply:
Following from above, one solution would be to edit [conf.py](https://github.com/scipy/scipy/blob/master/doc/source/conf.py#L323-L331) so that `numpy:` points to https://numpy.org/devdocs/.  I wouldn't know what problems that might cause, but it would allow `numpy:` interlinks to behave as intended.

Reply:
Reverting, as above.